tutorials/s4sdk-caching/s4sdk-caching.md

```diff
@@ -51,268 +56,136 @@ If each of these options apply to your use case, then we highly recommend the ca
 
 [ACCORDION-BEGIN [Step 3: ](Caching Command with SAP Cloud SDK)]
 
-The command cache allows parallel execution and asynchronous computation for efficient programming practices. Stored information is organized as a local key-value store. For each unique key the same response is expected in case of a "cache hit". Otherwise the cache response will be computed.
+The caching command allows parallel execution and asynchronous computation for efficient programming practices.
```

Comment:
The term caching command (also in the heading) no longer applies in version 3. Also, parallelization is not in focus here.